includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -197,6 +197,10 @@ public function __construct( $wikiId, $lang = self::DEFAULT_LANG ) {
 		$this->lang = $lang;
 	}
 
+	public function setHrefs( $hrefs ) {
```

Comment:
Not really, it's used outside of this class. We could trick it with Reflection although I don't like this approach...